CondFormats/JetMETObjects/src/METCorrectorParameters.cc

```diff
@@ -290,72 +176,141 @@ void METCorrectorParameters::printScreen() const
   std::cout<<"Parametrization Formula:       "<<definitions().formula()<<std::endl;
   std::cout<<"--------------------------------------------"<<std::endl;
   std::cout<<"------- Bin contents -----------------------"<<std::endl;
-  for(unsigned i=0;i<size();i++)
-    {
-      for(unsigned j=0;j<definitions().nBinVar();j++)
-        std::cout<<record(i).xMin(j)<<" "<<record(i).xMax(j)<<" ";
-      std::cout<<record(i).nParameters()<<" ";
-      for(unsigned j=0;j<record(i).nParameters();j++)
-        std::cout<<record(i).parameter(j)<<" ";
-      std::cout<<std::endl;
-    }
+  for(unsigned i=0;i<size();i++) //mRecords size
+  {
+    std::cout<<record(i).MetAxis()<<"  ";
+    std::cout<<"nBinVar ("<<definitions().nBinVar()<<")  ";
+    for(unsigned j=0;j<definitions().nBinVar();j++)
+      std::cout<<record(i).xMin(j)<<" "<<record(i).xMax(j)<<" ";
+    std::cout<<"nParameters ("<<record(i).nParameters()<<") ";
+    for(unsigned j=0;j<record(i).nParameters();j++)
+      std::cout<<record(i).parameter(j)<<" ";
+    std::cout<<std::endl;
```

Comment:
@cvuosalo The "cout" is inside "printScreen" method which is made for printout on screen.
These are not used in the general process of the class purpose. Especially when i make db files by local run. Also JEC db maker also uses similar method. Please consider this.